seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -1716,6 +1717,7 @@ def test_sv_wgs_get_es_variants(self):
         self.assertListEqual(variants, [PARSED_SV_WGS_VARIANT])
 
         self.assertExecutedSearch(filters=[
+            {'terms': {'rsid': ['rs9876']}},
```

Comment:
Does it also need a separated test for the `variant_ids`?

Reply:
no I checked and we are already testing that behavior  in other existing tests, but good catch